README.md

```diff
@@ -178,8 +177,8 @@ like the native ES5 `JSON` object.
 
 To use from Node:
 
-```
-npm install json5
+```sh
+$ npm install json5
```

Comment:
@gasi I fixed them in post rather than wait for the submitter to fix. See cb2613e52523421a74e343c455ae463794761d1a.